<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Presenter</class>
 <widget class="QWidget" name="Presenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonOpen">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter number of company for prediction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinCompanyNumber"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonPredict">
       <property name="text">
        <string>Predict</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editPrediction">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
